vscode/package.json

```diff
@@ -285,11 +280,11 @@
         "icon": "$(play)"
       },
       {
-        "command": "qsharp-vscode.debugEditorContentsWithCircuit",
-        "title": "Debug Q# File with Circuit Diagram",
+        "command": "qsharp-vscode.runEditorContentsWithCircuit",
+        "title": "Run file and show circuit diagram",
         "category": "Debug",
```

Comment:
This should probably just be in the `Q#` category now.

Reply:
I was on the fence about this. It behaves almost exactly the same as "Run Q# File", also in the Debug category. The internals are the same and the output goes to the Debug Console.

On the other hand, it's also a lot like "Run File and Show Histogram", in that it just runs a whole simulation and shows the output... so I can see putting it into the same category as that one.

Even before this change, the similarity of "Debug: Run Q# File" and "Q#: Run File and Show Histogram" always bugged me a little bit, and this one just kind of overlaps with both.